src/main/java/org/team199/robot2020/RobotContainer.java

```diff
@@ -58,13 +62,16 @@
     private final Climber climber = new Climber();
     private final RobotPath[] paths;
     private final LinearInterpolation linearInterpol;
+    private final UsbCamera camera1 = MotorControllerFactory.configureCamera(Constants.Drive.kCamera1Port);
+    private final VideoSink cameraServer = CameraServer.getInstance().getServer();
```

Comment:
This could be moved to whichever subsystem is going to manage the cameras (existing Drivetrain or new CameraSubsystem).